src/main/java/com/elasticpath/tools/mavenminimal/reactor/MavenDriver.java

```diff
@@ -125,7 +126,8 @@ private static CommandLine determineMavenCommand(final Reactor reactor, final Li
 		}
 
 		if (reactor.isSingleThread()) {
-			mavenArguments.removeIf(s -> s != null && s.matches("-T.*"));
+			removeThreadingFlags(mavenArguments);
+			mavenArguments.add("-T1");
```

Comment:
Is this necessary?

Reply:
I'm not 💯 , but this felt it made things very explicit: it forces a single-thread, rather than relying on Maven to do just do that.